builder-support/debian/authoritative/debian-buster/README.source

```diff
@@ -1 +1,58 @@
-See /usr/share/doc/quilt/README.source
```

Comment:
technically, none of this is true for the PowerDNS builds of the package. Although I'm unsure if it warrants the deviation from downstream.

Reply:
In Debian I've added a completely new README.source; none of that in there is correct for the upstream builds. I'd keep it out for now.

Reply:
The new file is really all wrong; I've removed it.